addon/components/au-file-upload.js

```diff
@@ -0,0 +1,147 @@
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { guidFor } from '@ember/object/internals';
+
+export default class AuFileUploadComponent extends Component {
+  @service() store;
+  @service() fileQueue;
+  @tracked uploadErrorData = [];
+
+  constructor() {
+    super(...arguments);
+    this.queueName = `${guidFor(this)}-fileUploads`;
+  }
+
+  ////////
+  // Properties which may be overridden
+  ////////
+  get uploadingMsg() {
+    return `Bezig met het opladen van ${this.fileQueue.files.length} bestand(en). (${this.fileQueue.progress}%)`;
```

Comment:
I think this isn't 100% correct. This will show the global upload progress state and the global amount of files being uploaded. Using the data from the Queue related to this components might be more correct? It probably doesn't really matter in practice though since you usually only use 1 of these components on a page.

Reply:
I looked into this, but I didn't immediately see a way to get the progress of a specific queue.

Reply:
I think that's a docs issue. The [Queue](https://github.com/adopted-ember-addons/ember-file-upload/blob/4104f784f43c764e91cf13d99e6e922f468031a2/addon/queue.js#L18) class uses the same [mixin](https://github.com/adopted-ember-addons/ember-file-upload/blob/4104f784f43c764e91cf13d99e6e922f468031a2/addon/mixins/with-files.js#L82) as the service. So I assume `this.queue.files.length` and `this.queue.progress` should work. I haven't tried it myself though.